cli/cmd/search.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/search"
+)
+
+// NewSearchCmd creates search command.
+func NewSearchCmd() *cobra.Command {
+	var searchCmd = &cobra.Command{
+		Use:   "search [PROGRAM]",
+		Short: "search available versions of tarantool/tt",
```

Comment:
`search` should be capitalized for the same style of help output,

Reply:
Fixed